hack/build/config.sh

```diff
@@ -62,3 +62,16 @@ function parseTestOpts() {
         esac
     done
 }
+
+function getClusterType() {
+	local image
+	case "${KUBEVIRT_PROVIDER}" in
+	"k8s-1.10.4")
```

Comment:
In how many different places will we need to change this when we upgrade to k8s-1.11 or even k8s-1.10.5?

Reply:
I am trying to confine them to the config.sh file. But it will also be in the automation directory. I don't think there are other places.